src/main/java/org/folio/tlib/util/TenantUtil.java

```diff
@@ -27,6 +27,10 @@ public static String tenant(RoutingContext ctx) {
       throw new IllegalArgumentException(
           "X-Okapi-Tenant header must match " + TENANT_PATTERN_STRING);
     }
+    if (tenant.length() > 31) {
+      throw new IllegalArgumentException(
+          "X-Okapi-Tenant header size must not be greater than 31");
+    }
```

Comment:
Okapi and PostgreSQL currently accept longer tenant ids.

The length restriction can be added later.

Reply:
OK